metabrainz/static/css/main.less

```diff
@@ -211,6 +211,26 @@ h1.page-title, h2.page-title {
   }
 }
 
+//team page 
+#team {
+  strong{ color: @green; }
```

Comment:
Looks like the other parts of main.less has a space between identifier and `{`. You should use the same style the rest of the file is using. ;)